client/gutenberg/extensions/presets/jetpack/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@automattic/jetpack-blocks",
-	"version": "6.8.0-beta.1",
+	"version": "6.8.0",
```

Comment:
before the release, i will surely make some changes to the preset. will that require a version bump again?  or it won't mater until we run the actual npm publish command

Reply:
Which release are you referring to? This is preparing a new version of the bundle to publish, but that's not linked to the Jetpack release cycle. Presumably, this would go out with Jetpack 6.7.1, and surely there will be a new bundle before the 6.8 release of Jetpack as well.

It seems like a good practice to publish the commit that bumps the version, but that's not a hard requirement. I can save the version bump for later if there are things you'd like to get in.

Reply:
> Which release are you referring to? 

I was referring to 6.8, because i assumed these had the same versioning as Jetpack.
If were we to update the bundles before the release, what would change the new bundle version to?

Reply:
Should we tag it `6.7.1`, since it will be released with the `6.7.1` release?
https://github.com/Automattic/jetpack/pull/10596